security/KeePassXC/Portfile

```diff
@@ -20,19 +20,21 @@ platforms               darwin
 license                 GPL-2+
 license_noconflict      openssl
 
-github.setup            keepassxreboot keepassxc 2.5.1
+github.setup            keepassxreboot keepassxc 2.5.4
 github.tarball_from     releases
 distname                keepassxc-${version}-src
 use_xz                  yes
 distfiles-append        ${distname}${extract.suffix}.sig
 
 # See keepassxc-${version}-src.tar.xz.DIGEST on upstream GitHub releases page for SHA256 sums
 checksums               ${distname}${extract.suffix} \
-                        rmd160  4b3c639ec001fec15f16560f795dbbfd9a79ab9c \
-                        sha256  ef33258b859a7b996af007113613b0f6210f2341e8f5fb3a005564262c2caf30 \
-                        size    6798880 \
+                        rmd160  635c4eff7769c3edf80af1e3070d535157487ee6 \
+                        sha256  a55e0801c318b02b1ac4e16e9b7a87ccfa7b039ea60d2c62610bd1bbbdd6cd4a \
+                        size    6839396 \
                         ${distname}${extract.suffix}.sig \
-                        size    488
+                        rmd160  44153573ac36bc47b97e58b845bfee2a91db0c6a \
+                        sha256  fc527280cfa05c2ed75529c280432ab8d301f69911f2d74c56fe8061f2b0d2f1 \
+                        size 488
```

Comment:
Add some spaces to make '488' align with the other checksum values.

Reply:
This will also trigger a new rebuild attempt.

Reply:
Done. And this time it worked.